### PR TITLE
if not set params of "-i", all traffic should not redirected default

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: souleen <zhuxiaoyang1996@gmail.com>

if not set "-i", all traffic should not redirected default.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[X] Networking
[ ] Performance and Scalability
[X] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Comment by soulseen** [comment]
> Not sure whether that's OK.
> In any case, IPv6 should be treated consistently with IPv4.
> And this change needs unit tests.

I have updated this PR, please review it again. Thanks a lot!

File: .gitignore [hunk 1]
```diff
@@ -52,3 +52,4 @@ out/
 etc/
 var/
 .idea/
+.DS_Store
```
Comment (rlenglet) [inline .gitignore:55]: Revert this. You can solve this problem on your system by setting a global .gitignore, cf. for example http://egorsmirnov.me/2015/05/04/global-gitignore-file.html

**Comment by soulseen** [comment]
> Please include a description of what bug/use case this is fixing

in fact, if the value of `NetworkRange ` not set, all traffic should not redirect and return to the upstream chain default.

**Comment by hzxuzhonghu** [comment]
That means if -i = "", the current behavior is traffic redirected to envoy, which is wrong.

**Pull request closed without merging**